gluetool/proxy.py

```diff
@@ -5,6 +5,8 @@
 # Proxy class comes from the AS recipe
 # http://code.activestate.com/recipes/496741-object-proxying/
 
+from builtins import object
```

Comment:
Why is this necessary?

Reply:
https://stackoverflow.com/questions/36517648/in-python-why-import-object-from-builtins-module)

Reply:
Ah, I see. Nice.